gallery/src/components/NxFormGroup/NxFormGroupPage.tsx

```diff
@@ -0,0 +1,128 @@
+import NxFormGroupRequiredExample from './NxFormGroupRequiredExample';
+import NxFormGroupSublabelExample from './NxFormGroupSublabelExample';
+import NxFormGroupRichLabelExample from './NxFormGroupRichLabelExample';
+import NxFormGroupExistingAriaExample from './NxFormGroupExistingAriaExample';
+
+const nxFormGroupExampleCode = require('!raw-loader!!./NxFormGroupExample').default,
+    nxFormGroupSublabelExampleCode = require('!raw-loader!!./NxFormGroupSublabelExample').default,
+    nxFormGroupRichLabelExampleCode = require('!raw-loader!!./NxFormGroupRichLabelExample').default,
+    nxFormGroupExistingAriaExampleCode = require('!raw-loader!!./NxFormGroupExistingAriaExample').default,
+    nxFormGroupRequiredExampleCode = require('!raw-loader!!./NxFormGroupRequiredExample').default;
+
+const NxFormGroupPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <p className="nx-p">
+        An <code className="nx-code">NxFormGroup</code> is a wrapper around a form field which provides the labels
+        and overall spacing. Most commonly, <code className="nx-code">NxFormGroup</code> wraps
+        an <code className="nx-code">NxTextInput</code>, but it may wrap other content such as
+        a <code className="nx-code">&lt;select&gt;</code>. It <em>should not</em> be used to wrap radio
+        and checkbox groups, as those are best encapsulated
+        in <code className="nx-code">&lt;fieldset&gt;</code> elements.
+      </p>
+      <NxTable>
+        <NxTableHead>
+          <NxTableRow>
+            <NxTableCell>Prop</NxTableCell>
+            <NxTableCell>Type</NxTableCell>
+            <NxTableCell>Required</NxTableCell>
+            <NxTableCell>Default</NxTableCell>
+            <NxTableCell>Details</NxTableCell>
+          </NxTableRow>
+        </NxTableHead>
+        <NxTableBody>
+          <NxTableRow>
+            <NxTableCell>label</NxTableCell>
+            <NxTableCell>string | ReactNode</NxTableCell>
+            <NxTableCell>Yes</NxTableCell>
+            <NxTableCell>N/A</NxTableCell>
+            <NxTableCell>JSX content to render as the label for the group. Must not be null or undefined.</NxTableCell>
+          </NxTableRow>
+          <NxTableRow>
+            <NxTableCell>sublabel</NxTableCell>
+            <NxTableCell>string | ReactNode</NxTableCell>
+            <NxTableCell>No</NxTableCell>
+            <NxTableCell>empty</NxTableCell>
+            <NxTableCell>JSX content to render as the sublabel for the group.</NxTableCell>
+          </NxTableRow>
+          <NxTableRow>
+            <NxTableCell>isRequired</NxTableCell>
+            <NxTableCell>boolean</NxTableCell>
+            <NxTableCell>No</NxTableCell>
+            <NxTableCell>false</NxTableCell>
+            <NxTableCell>
+              Sets whether the input should display the optional flag – the flag is present by default and
+              setting <code className="nx-code">isRequired</code> to true removes the flag.
```

Comment:
I wonder if we should default to "required". I know that most of the fields in IQ are required, and Michael Martz thinks RM is probably ~50/50

Reply:
> I wonder if we should default to "required". I know that most of the fields in IQ are required, and Michael Martz thinks RM is probably ~50/50

I asked myself that, but I decided to go this way because it would be consistent with the following two things:

1. `NxTextInput` defaults to not required in the sense that if you want it to validate that it isn't empty, you have to go out of your way to supply a validator function
2. HTML `<input>`s have a `required` attribute which you have to add if you want them to be considered required by HTML form validation